src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -10,7 +10,7 @@
     <x>0</x>
     <y>0</y>
     <width>504</width>
-    <height>678</height>
+    <height>890</height>
```

Comment:
Is the dialog in the screenshot at the default size? If yes, I'm not sure we need all the empty space at the bottom.

Reply:
Maybe it's to accommodate devmode options? But I guess we should tailor the size to non-devmode.

Reply:
So yes, this is to accommodate devmode options, which we also had done previously. I am totally fine with tailoring to non-devmode.

Reply:
Ah, I wasn't sure. I think it depends quite a bit on display resolution or scaling.

Reply:
Probably...